spec/index.html

```diff
@@ -11799,6 +11799,58 @@ <h2>Internet Media Type, File Extension and Macintosh File Type</h2>
       </div>
     </section>
     
+    <section id="changes-1-1" class="appendix informative">
+      <h2>Changes between SPARQL 1.1 Query Language and SPARQL 1.2 Query Language</h2>
+      <ul>
+        <li>
+            Fix errata:
+            <ul>
+              <li><a href="https://www.w3.org/2013/sparql-errata#editorial-query-1">editorial-query-1</a>: Missing right parenthesis in <a href="#defn_evalGraph">Evaluation of Graph definition</a></li>
+              <li><a href="https://www.w3.org/2013/sparql-errata#editorial-query-2">editorial-query-1</a>: Missing space in <a href="#defn_algJoin">Join definition</a></li>
```

Comment:
editorial-query-1 -> editorial-query-2
```suggestion
              <li><a href="https://www.w3.org/2013/sparql-errata#editorial-query-2">editorial-query-2</a>: Missing space in <a href="#defn_algJoin">Join definition</a></li>
```